### PR TITLE
feat: can specify available UI languages for an organization

### PR DESCRIPTION
**Pull request by leo220yuyaodog**

### CODE REVIEW

File: web/src/OrganizationEditPage.js [hunk 1]
```diff
@@ -255,6 +255,30 @@ class OrganizationEditPage extends React.Component {
...
+            {Setting.getLabel(i18next.t("Organization:Languages"), i18next.t("Organization:Languages - Tooltip"))} :
+          </Col>
+          <Col span={22} >
+            <Select virtual={false} mode="tags" style={{width: "100%"}}
+              value={this.state.organization.languages}
+              onChange={(value => {
+                this.updateOrganizationField("languages", value);
+              })} >
+              {[
+                {value: "zh", label: "简体中文"},
```
Comment (hsluoyz) [inline web/src/OrganizationEditPage.js:269]: Use exsiting order: `["en", "zh", "es", "fr", "de", "ja", "ko", "ru"]`, not alphabet

File: web/src/SelectLanguageBox.js [hunk 2]
```diff
@@ -28,28 +28,44 @@ class SelectLanguageBox extends React.Component {
     super(props);
     this.state = {
       classes: props,
+      languages: props.languages ?? ["en", "zh", "es", "fr", "de", "ja", "ko", "ru"],
     };
   }
 
+  items = [this.getItem("English", "en", flagIcon("US", "English")),
```
Comment (hsluoyz) [inline web/src/SelectLanguageBox.js:35]: new line after [

File: web/src/locales/de/data.json [hunk 3]
```diff
@@ -1,4 +1,8 @@
 {
+  "Organization": {
```
Comment (hsluoyz) [inline web/src/locales/de/data.json:2]: organization, don't create new category

File: web/src/App.js [hunk 4]
```diff
@@ -636,7 +636,7 @@ class App extends Component {
                 {
                   this.renderAccount()
                 }
-                <SelectLanguageBox />
+                {this.state.account && <SelectLanguageBox languages={this.state.account.organization.languages} />}
```
Comment (hsluoyz) [inline web/src/App.js:639]: The user is not logged in yet. Use `application.organizationObj`

**Review by hsluoyz — CHANGES_REQUESTED** [review]

File: web/src/App.js (hunk 4 above)
Comment (leo220yuyaodog) [reply web/src/App.js:639]: A &&  B is equel 

`
if ( A !== "null" ||  A !== undefined){
    B
}
`

And the `<SelectLanguageBox />` is after logged in.

**Review by hsluoyz — APPROVED** [review]

**Pull request merged**